savant/deepstream/nvinfer/element_config.py

```diff
@@ -411,14 +414,36 @@ def process(self, msg, kwargs):
     nvinfer_config['property']['operate-on-class-ids'] = parent_class_id
 
     # register the model
-    object_labels = {}
-    if issubclass(model_type, ObjectModel):
-        object_labels = {
-            obj.class_id: obj.label for obj in model_config.output.get('objects')
-        }
-    model_uid = register_model_objects(
-        element_config.name, object_labels, RegistrationPolicy.Override
-    )
+    if issubclass(model_type, (ObjectModel, ComplexModel)):
+        output_objects = model_config.output.get('objects', [])
+        if output_objects:
+            logger.debug('Registering output objects for the model ...')
```

Comment:
Try to name the model here, in logger.debug.

Reply:
It's already included in all messages in this function through the logging adapter defined above.